### ISSUE TITLE
PDF/UA 7.3 test 1 and WCAG 4.1.2 test 20 false negatives

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#77]
[This very small PDF document](https://github.com/veraPDF/veraPDF-wcag-algs/files/6970190/test-document-12.pdf) contains an image with no alternative text. Below the image is a figure caption which is tagged as a normal paragraph. This file should generate the following two errors:
- PDF/UA 7.3 test 1 "Missing alternative (or replacement) text in the Figure"
- WCAG 4.1.2 test 20 "Caption is incorrectly tagged as a paragraph"

But the PDF checker doesn't detect either one of them, so these qualify as false negatives.

**Comment by bdoubrov** [comment]
The figure does have an alternative text "P3#yIS1" (a but weird one, but non-empty).

The caption is not recognized as the Figure is immediately followed by a space, which separates it from the following real caption. We'll check if such space-only chunks can be ignored in recognition.

**Comment by jzuidweg** [comment]
In the Word source document the figure had no alt-text so it seems that the PDF export inserted a random alt-text. In this case the checker was right that there is in fact an alt-text. In a future release we may consider implementing an AI based check to see whether an alt-text is meaningful. This is not easy to check, but random strings like these or file names (which are also sometimes inserted automatically as alt-text) should be possible to catch.

Thanks for picking up the issue with the figure caption!
